closures.js

```diff
@@ -221,13 +240,13 @@ var allPotentialFriends = "?";
 /****** INSTRUCTIONS PROBLEM 8 ******/
 /* Here we have a for loop that will iterate as long as i is less than or equal
 to 5. What we need to do is console.log(i) so that it logs like so:
- 1 second after call - log 0
- 2 seconds after call - log 1
- 3 seconds after call - log 2
- 4 seconds after call - log 3
- 5 seconds after call - log 4
- 6 seconds after call - log 5
- However, because each call to logCounter occurs after the loop has finished,
+ immediatly - log 0
+ 1 seconds after call - log 1
+ 2 seconds after call - log 2
+ 3 seconds after call - log 3
+ 4 seconds after call - log 4
+ 5 seconds after call - log 5
+ However, because each call to setTimeout occurs after the loop has finished,
```

Comment:
Maybe change it to "each call to console.log occurs after the loop has finished" for more accuracy